ui/app/components/variable-form.js

```diff
@@ -206,6 +208,7 @@ export default class VariableFormComponent extends Component {
         destroyOnClick: false,
         timeout: 5000,
       });
+      this.removeExitHandler();
```

Comment:
Don't bother the user when transitioning because of a successful save

Reply:
Hey, buddy! Looks like we end up removing the event listener twice and that causes our tests to fail.

The bottom of this [blog](https://balinterdi.com/blog/warning-about-losing-unsaved-changes-in-ember-octane/) should have a quick patch.